modules/core/src/main/java/org/apache/ignite/internal/hlc/HybridClockImpl.java

```diff
@@ -72,31 +63,40 @@ public final long nowLong() {
     }
 
     @Override
-    public final long currentLong() {
-        long current = currentTime();
+    public final long nowLong(HybridTimestamp causal) {
```

Comment:
Why do you rewrite the logic instead of reuse HLC#update method? Moreover the update method is better, because more frequently use increment.

Reply:
Yes, it seems I unintentionally re-wrote update method.
Removed these methods and switched to existing "update".